docs/kit/operation-view/page04_local_setup_dataplane.md

```diff
@@ -0,0 +1,98 @@
+# Setting up a local EDC Data Plane
+
+## Basics
+
+The EDC is split into control and data plane.
+The data plane handles the actual data transfer between parties.
+The control plane manages the following:
+
+- Resource Management (e.g. Assets, Policies & Contract Definitions CRUD)
+- Contract Offering & Contract Negotiation
+- Data Transfer Coordination / Management
+
+The EDC data plane can run as a single container on your local machine.
+The following is a short overview of the necessary steps to start up the default configuration.
+
+## Building
+
+TractusX EDC is build with Gradle. The following command creates the default data plane as a docker container:
+
+```shell
+./gardlew :edc-dataplane:edc-dataplane-hashicorp-vault:dockerize
+```
+
+## Example Configuration
+
+The following commands can be used to create the necessary configuration files for the EDC container.
+They assume sane - but unsafe - defaults. An explanation of the respective parameters can be found [here](https://github.com/eclipse-tractusx/tractusx-edc/blob/main/edc-dataplane/edc-dataplane-hashicorp-vault/README.md).
+
+::: Caution
+
+The following configuration is for testing purposes only. Do not use it in production.
+:::
+
+### Example configuration.properties
```

Comment:
Please double check the configs with the migration guide. Also regarding the observation jetty context

Reply:
@florianrusch-zf let's have a short call. THX